src/BuildingRegistry.Api.BackOffice/BuildingRegistry.Api.BackOffice.csproj

```diff
@@ -36,6 +36,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\BuildingRegistry.Api.BackOffice.Handlers\BuildingRegistry.Api.BackOffice.Handlers.csproj" />
```

Comment:
don't we need a dependency on Abstractions as well?

Reply:
Not really, it is automatically added (somewhere in .net versions).